tools/settings_template.py

```diff
@@ -33,7 +33,8 @@
 
 JAVA = 'java' # executable
 
-# CLOSURE_COMPILER = '..' # define this to not use the bundled version
+# To install Closure compiler, run 'npm install' in Emscripten root directory
+CLOSURE_COMPILER = os.path.join(EMSCRIPTEN_ROOT, 'node_modules', '.bin', 'google-closure-compiler' + ('.cmd' if os.name == 'nt' else ''))
```

Comment:
Can you simplify this to avoid to be just  `CLOSURE_COMPILER = os.path.expanduser(os.getenv('CLOSURE_COMPILER', '{{{ CLOSURE_COMPILER }}})`.

Actually just `CLOSURE_COMPILER = {{{ CLOSURE_COMPILER }}}`.

We now have generic mechanism for override these settings using env var so we don't need to it manually anymore.  (Basically you can set EM_<VAR> to override anything in settings file, see comment at start of this file).